drivers/java/src/main/java/com/rethinkdb/net/Connection.java

```diff
@@ -345,7 +345,7 @@ public void runNoReply(ReqlAst term, OptArgs globalOpts) {
 
     void stop(Cursor cursor) {
         // The server does reply to a stop request, though the response doesn't have a value.
-        runQuery(Query.stop(cursor.token));
+        runQueryNoreply(Query.stop(cursor.token));
```

Comment:
Someone changed this and i just change back. This method is only used by Cursor.java currently.